src/onnx/parse_randomuniform_ops.cpp

```diff
@@ -35,8 +35,7 @@ namespace onnx {
 
 struct parse_randomuniform_ops : op_parser<parse_randomuniform_ops>
 {
-    const std::set<shape::type_t> valid_types = {
-        shape::float_type, shape::half_type, shape::double_type};
+    std::set<shape::type_t> valid_types = {shape::float_type, shape::half_type, shape::double_type};
```

Comment:
This should be static const.

Reply:
Yes, but i have to make it `static const inline` then.

Reply:
And then, it complains about : warning: declaration requires a global destructor [-Wglobal-constructors]
    static const inline std::set<shape::type_t> valid_types = {shape::float_type, shape::half_type, shape::double_type};